pyplanet/apps/contrib/music_server/__init__.py

```diff
@@ -13,8 +13,8 @@
 
 
 class MusicServer(AppConfig):
-	game_dependencies = ['trackmania', 'shootmania']
-	app_dependencies = ['core.maniaplanet', 'core.pyplanet']
+	game_dependencies = ['trackmania', 'shootmania', 'trackmania_next']
+	app_dependencies = ['core.maniaplanet', 'core.trackmania', 'core.pyplanet']
```

Comment:
@TheMaximum why depend on core.trackmania? This will make the app not load in Shootmania as the core.trackmania isn't loaded there.